### DISCUSSION TITLE
Meeting notes 4/12

### DISCUSSION THREAD

**Discussion by wavingtowaves** [SORTEE-Github-Hackathon/manuscript#150] [General]
### General notes
@emmajhudgins @BrandonEdwards @Aariq @DrMattG
How does 'hypothesis' deal with updates to the manuscript -- TBD
When you're making edits, make them on your own **branch**.

**Next 2 days**
- [x] 1a. Rob drafts paragraph on Github for lab notebook. (Rob)
- [x] 1b. Emma drafts paragraph on using GitHub for manuscript. (Emma)
- [x] 2. Review paragraph PRs

**Next week**
- [x] 3. Can we resolve comments that are left using hypothesis? Sort of. See comments by @Aariq below (Rob)
- [x] 4. Fix all the little issues: formatting, citations that don't work (Brandon, Rob). As we do this, scan through hypothesis and indicate that we've resolved that particular comment. **From April 18-20**
- [x] 5. Fixing all the comments left in hypothesis (if any comment left in hypothesis needs more discussion, turn into an issue) (Emma, Matt) **April 20 and beyond**
- [x] 6. Put all figures/tables into manuscript in the new section. (Eric)

**By Thursday May 5th**
- [ ] 9. read through for content. Leave comments using 'hypothesis'. Indicate where figures should be referenced (All)

**By Thursday May 12**
- [ ] 10. Incorporate group feedback into manuscript (Eric, Matt, Brandon)

**By Thursday May 19th**
- [ ] 11. Final final read through for resolving inconsistencies (Rob)

**Comment by wavingtowaves** [comment]
So, it seems like the annotations are only able to be removed by the people who originally created the annotations: https://web.hypothes.is/help/deleting-an-annotation/

I don't think this necessarily means we should avoid using it for our big read through of the manuscript, but it will require people delete their annotations when it's time to clean up the HTML.  What do you all think?

We can indeed us the 'reply' button to still indicate when we've addressed the comments.

**Reply by Aariq** [reply]
Yeah, and at some point if we really like using it and want to start fresh, we can modify the github action to publish the manuscript at a different URL to get a fresh start.

**Reply by Aariq** [reply]
Or what about creating a "private group"?  Will that allow for a different set of comments?

![Screen Shot 2022-04-12 at 5 00 24 PM](https://user-images.githubusercontent.com/25404783/163053818-9a74fefe-4fbc-49d2-9be0-c94c578102a1.png)

**Reply by wavingtowaves** [reply]
Maybe private group, especially just for this new round of edits that our smaller group is doing?

**Reply by DrMattG** [reply]
It makes sense to make a private group for editing

**Reply by Aariq** [reply]
Ok, here's an invite to a private group: https://hypothes.is/groups/oVKrjvLv/big-picture
I think what happens is you click on that link to join the group, then you'll go to the [rendered manuscript](https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html) and switch hypothes.is to the private group:
![Screen Shot 2022-04-15 at 11 16 22 AM](https://user-images.githubusercontent.com/25404783/163588444-bfb4498b-2b02-479c-a45d-0b9392f3cbaf.png)


@robcrystalornelas @emmajhudgins @BrandonEdwards @DrMattG

**Comment by wavingtowaves** [comment]
That worked great for me @Aariq. Thanks for setting it up!

**Comment by Aariq** [comment]
Just FYI, I'll be out of town from April  27th to May 1st.  I think the timing will work out fine, but don't wait on me for approving PRs or making edits this week.

**Comment by emmajhudgins** [comment]
I think we've made it to step 9 now, notwithstanding the possibility of a few lingering hypothesis comments that Matt and I missed. I believe this means that all of us can start a content read through using the new hypothesis group! @robcrystalornelas @DrMattG @BrandonEdwards @Aariq (after your trip!)

**Reply by wavingtowaves** [reply]
Really great work @emmajhudgins and @DrMattG. And yes, timing will work out totally fine @Aariq, just let us know if you need a few more days after May 5th to review.
